src/backend/core/api/viewsets.py

```diff
@@ -243,6 +243,7 @@ class TeamViewSet(
     permission_classes = [permissions.AccessPermission]
     serializer_class = serializers.TeamSerializer
     queryset = models.Team.objects.all()
+    lookup_field = 'slug'
```

Comment:
nit: double quote

Reply:
I removed this lookup field as we'll keep on using ids for URIs